### PR TITLE
[MeshMoving] cleanup 

### PR DESCRIPTION
**Pull request by philbucher**
This PR makes some final cleanup such that the MeshMovingApp can become a core-app

A readme is added (@rubenzorrilla do you want to add your contributions?)
Also the old ball-vertex mesh-solver is removed

### CODE REVIEW

File: applications/MeshMovingApplication/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## MeshMoving Application
+
+The MeshMoving Application (formerly ALEapplication) contains different mesh-solving techniques.
```
Comment (adityaghantasala) [inline applications/MeshMovingApplication/README.md]:
```suggestion
The MeshMoving Application (formerly ALEapplication) contains different mesh-update techniques.
```

**Comment by rubenzorrilla** [comment]
> This PR makes some final cleanup such that the MeshMovingApp can become a core-app
> 
> A readme is added (@rubenzorrilla do you want to add your contributions?)
> Also the old ball-vertex mesh-solver is removed

Yes! I'll do it ASAP.

**Comment by philbucher** [comment]
Ping @rubenzorrilla

**Comment by philbucher** [comment]
@rubenzorrilla I suggest that I merge this and then you can add explanations for your things in a separate PR

**Review by AndreasWinterstein — APPROVED** [review]

**Pull request merged**